src/main/resources/home.html

```diff
@@ -0,0 +1,36 @@
+<!--// use Mustaches tags to conditionally show the markup in the screenshot provided..-->
+<!--if userName does not exist, show the login form like screenshot number 1-->
+<!--if userName does exist, show the message submission form and message list like screenshot number 2-->
+
+<html>
+<body>
+{{#name}}Hello, {{.}}!{{/name}}
+
+{{^name}}
+<form action="/login" method="post">
+    <input type="text" placeholder="Enter your name" name="userName"/>
+    <button type="submit">Login</button>
+</form>
+{{/name}}
+
+{{#message}}
+<form action="/add-message" method="get">
+    <input type="text" placeholder="Add your message" name="insertMessage"/>
+    <button type="submit">Message</button>
+</form>
+{{/message}}
+
+
+<form action="/delete-message" method="post">
+<li>
+{{text}}<input type="hidden" value="{{id}}" name="deleteMessage"/>
+    <button type="submit">Delete</button>
+</li>
+</form>
+</body>
+</html>
+
+<!--in "home.html" add a button or link called "Delete" next to each message-->
+<!--it will need to send the id as a parameter-->
+<!--ASK WHATS HIS FACE MIKE!! To explain-->
+
```

Comment:
Looks like you did exactly what was needed.  Let me know if you need anymore explaining.